src/components/volume-control/VolumeBar.js

```diff
@@ -115,8 +115,8 @@ class VolumeBar extends Component {
         onPercentageChange={this.handlePercentageChange}
         stepForward={this.stepForward}
         stepBack={this.stepBack}
-        className={classNames(className, 'video-react-volume-bar video-react-slider-bar')}
         {...this.props}
```

Comment:
When `this.props.className` is defined,  it overwrites className applied on line 118.